supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala

```diff
@@ -47,8 +51,32 @@ sealed trait ConfigAsyncJobExecutionActor extends SharedFileSystemAsyncJobExecut
   def writeTaskScript(script: Path, taskName: String, inputs: WorkflowCoercedInputs): Unit = {
     val task = configInitializationData.wdlNamespace.findTask(taskName).
       getOrElse(throw new RuntimeException(s"Unable to find task $taskName"))
-    val inputsWithFqns = inputs map { case (k, v) => s"$taskName.$k" -> v }
-    val command = task.instantiateCommand(task.inputsFromMap(inputsWithFqns), NoFunctions).get
+
+    val taskDefinition = task.womDefinition.toTry.get
+
+    // Build WOM versions of the provided inputs by correlating with the `InputDefinition`s on the `TaskDefinition`.
+    val providedWomInputs: WomEvaluatedCallInputs = (for {
+      input <- inputs.toList
+      (localName, womValue) = input
+      womInputDefinition <- taskDefinition.inputs
+      if localName == womInputDefinition.localName.value
+    } yield womInputDefinition -> womValue).toMap
+
+    // TODO WOM FIXME
+    // This may force `none`s onto potentially initialized optionals. This is currently required because otherwise
+    // expressions using uninitialized optionals explode on evaluation.
+    val optionalsForciblyDefaultedToNone = for {
+      optional <- taskDefinition.inputs.collect { case oid: OptionalInputDefinition => oid }
+      // Only default to `none` if there was no input value explicitly provided.
+      if !inputs.contains(optional.localName.value)
+    } yield optional -> WomOptionalValue.none(optional.womType.memberType)
```

Comment:
Would love to chat IRL about what's missing here. These config-specified-tasks are usually relatively simple, but  true they do often [contain optionals](https://github.com/broadinstitute/cromwell/blob/5e325e98785670d2bf12bd98590a082c89e25a94/cromwell.examples.conf#L250-L257).

Reply:
Discussed IRL. I'm going to leave the code effectively as it is now with a ticket and more comments.